code/modules/surgery/eye_cybernetic_implant.dm

```diff
@@ -0,0 +1,31 @@
+/datum/surgery/eye_cybernetic_implant
+	name = "eye cybernetic implant"
+	steps = list(/datum/surgery_step/incise, /datum/surgery_step/clamp_bleeders, /datum/surgery_step/retract_skin, /datum/surgery_step/cybernetic_implant, /datum/surgery_step/fix_eyes, /datum/surgery_step/close)
+	location = "eyes"
+
+/datum/surgery_step/cybernetic_implant
+	accept_any_item = 1
```

Comment:
That's the point xxalpha, this SURGERY can be eyes only, but this step, if you make the changes I suggested, can be used for any and all future cybernetic_implants.